boards/arm/black_f407_generic/black_f407ve_defconfig

```diff
@@ -24,6 +24,9 @@ CONFIG_CLOCK_CONTROL=y
 # Enable GPIO
 CONFIG_GPIO=y
 
+# Enable SPI
+CONFIG_SPI=y
```

Comment:
It is up to application to control the activation of peripherals (aside from basic ones: clocks, gpio, serial for console) 
This should be done on sample side (`samples/drivers/spi_flash/prj.cnf`)

Reply:
OK